spec/features/user_can_update_length_of_rental_while_status_is_out_spec.rb

```diff
@@ -1,5 +1,5 @@
 require 'rails_helper'
-RSpec.describe 'User', type: :feature do
+RSpec.describe 'User', type: :feature, js: true do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.